platform/broadcom/sonic-platform-modules-accton/as7726-32x/modules/pddf_custom_psu.c

```diff
@@ -0,0 +1,204 @@
+#include <linux/module.h>
+#include <linux/jiffies.h>
+#include <linux/i2c.h>
+#include <linux/hwmon.h>
+#include <linux/hwmon-sysfs.h>
+#include <linux/err.h>
+#include <linux/delay.h>
+#include <linux/mutex.h>
+#include <linux/sysfs.h>
+#include <linux/slab.h>
+#include <linux/dmi.h>
+#include "../../../../pddf/i2c/modules/include/pddf_psu_defs.h"
```

Comment:
there are too many pddf directory in the sonic-buildimage, we should add the include path in the search path and build.

Reply:
@jostar-yang 
Can you take care of this comment?